### DISCUSSION TITLE
✨ [Feature]: 保障组件的API/Demo遵循规范

### DISCUSSION THREAD

**Discussion by TinsFox** [DevCloudFE/vue-devui#1369] [General]
This issue provides visibility into API & Demo discussions and their statuses. 

## Basic 基础组件

- [x] #72 
- [x] #76 

## Layout
- [ ] #112 
- [ ] #111 
- [ ] #113 

## Navigation 导航组件
- [ ] #78 

## Feedback 反馈组件
- [ ] #88 
- [ ] #86 

## Data Entry 数据录入
- [ ] #118 
- [ ] #92 
- [ ] #90 
- [x] #82 

## Data Display 数据展示
- [ ] #125 
- [ ] #122 
- [ ] #117 
- [ ] #115 
- [ ] #114 
- [ ] #84 
- [ ] #80

**Comment by TinsFox** [comment]
@xingyan95 讨论可以在这里关联一下，做个入口分类找起来会方便点。Discussions 的使用率目前还是会偏低一点

**Comment by kagol** [comment]
api 命名方式应该统一成中划线风格，目前1.0版本的40个组件还有以下组件需要优化下（小驼峰改成中划线分隔的烤串风格）：
1. Pagination
2. Loading
3. ReadTip
4. AutoComplete
5. Checkbox
6. EditableSelect
7. Input
8. Radio
9. Slider
10. Switch
11. TagInput
12. Textarea
13. Avatar
14. Countdown
15. ImagePreview
16. Progress
17. Rate
18. Splitter

**Comment by kagol** [comment]
开发规范以贡献指南中的开发规范文档为准：
[https://vue-devui.github.io/contributing/](https://vue-devui.github.io/contributing/)

规范覆盖不到的部分，欢迎在评论区踊跃参与讨论！

**Comment by kagol** [comment]
2022.4.20 updates

已优化以下组件文档：
- Alert
- AutoComplete
- Button
- Card
- Checkbox
- EditableSelect
- Input
- Loading
- Modal
- Notification
- Overlay
- Pagination
- Panel
- Radio
- ReadTip
- Result
- Search
- Status

**Comment by kagol** [comment]
2022.4.20 updates

已优化以下组件文档：

- Slider
- Switch
- Textarea
- Upload
- Avatar
- Badge
- Card
- Comment
- Countdown
- ImagePreview
- Progress
- Rate
- Skeleton
- Statistic
- Tag
- Timeline
- Tree
- Grid
- Tree
- Grid
- Layout
- Splitter
